lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-    "packages": ["packages/*", "suite-native/*", "scripts"],
+    "packages": ["packages/*", "suite-native/*", "suite-common/*", "scripts"],
```

Comment:
So if it was decided to have separate folders for desktop and mobile in the root, we will also have a separate folder for connect? It should be `suite`, `suite-native`, `suite-common`, `connect` and packages? I just feel that the packages are getting slightly out of hand 😅

Reply:
It will be something like that, in the end we should have about 5 or 6 root folders with packages. We already came to some agreement about packages with other team leads. We are currently working on article that will explain everything, maybe even today, then we can discuss this on code improvements.

For now we will have just suite-native and suite-common. Rest of them is not necessary and not important now.